dapp/views/receive/receive.jsx

```diff
@@ -2,15 +2,38 @@ import React       from 'react'
 import { connect } from 'react-redux'
 import QRCode      from './qr_code'
 import BackCopy    from './back_copy'
+import coinbase    from '@poplocker/react-ui/assets/coinbase.svg'
+import shapeshift  from '@poplocker/react-ui/assets/shapeshift.svg'
+import forward     from '@poplocker/react-ui/assets/back.svg'
```

Comment:
Instead of adding new file, we can use arrow from send or receive button and turn it 90 degrees.

Reply:
Done